File: PadariaDoManoel/Dominio/Entities/UnidadeMedida.cs
using System;


namespace PadariaDoManoel.Entities
{
    class UnidadeMedida
    {
        private int idUnidadeMedida;
        private string descricao;

        public UnidadeMedida(int idUnidadeMedida, string descricao)
        {
            this.IdUnidadeMedida = idUnidadeMedida;
            this.Descricao = descricao;
        }

        public int IdUnidadeMedida { get => idUnidadeMedida; set => idUnidadeMedida = value; }
        public string Descricao { get => descricao; set => descricao = value; }
    }
}

File: PadariaDoManoel/src/main/java/Classes/UnidadeMedida.java
package Classes;

import javax.persistence.Column;
import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;
import javax.persistence.Table;


/**
 *
 * @author Gilson
 */

@Entity
@Table (name = "TB_UnidadeMedida")
public class UnidadeMedida {
    
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    @Column (name = "IDUnidadeMedida")
    private int idUnidadeMedida;
    @Column (name = "Descricao")
    private String descricao;

    public UnidadeMedida() {
    }

    public UnidadeMedida(int idUnidadeMedida, String descricao) {
        this.idUnidadeMedida = idUnidadeMedida;
        this.descricao = descricao;
    }

    public int getIdUnidadeMedida() {
        return idUnidadeMedida;
    }

    public void setIdUnidadeMedida(int idUnidadeMedida) {
        this.idUnidadeMedida = idUnidadeMedida;
    }

    public String getDescricao() {
        return descricao;
    }

    public void setDescricao(String descricao) {
        this.descricao = descricao;
    }
    
    
    
    
    
}
